services/web-server/src/resolvers/WorkerManager.js

```diff
@@ -1,4 +1,15 @@
+const { splitWorkerPoolId } = require('taskcluster-worker-manager/src/util');
```

Comment:
Oh, I missed this! [We don't import between microservices](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/microservices.md#independence).  I think it would be best to just copy this utility, since it's short and straightforward.  If it were more complex, the fix would be to create a new tc-lib-foo library.